pkg/manager/manager.go

```diff
@@ -511,7 +520,15 @@ func (s *FraktiManager) ListImages(ctx context.Context, req *kubeapi.ListImagesR
 	// if there is different in two sides, print the different if log lever is high enough
 	if glog.V(5) && len(imageMapList[0]) != len(intersectList) {
 		glog.Infof("Image black hole in %s:\n%v", imageServiceList[0].ServiceName(), getImageListDifference(imageMapList[0], imageMapList[1]))
-		glog.Infof("Image black hole in %s:\n%v", imageServiceList[0].ServiceName(), getImageListDifference(imageMapList[1], imageMapList[0]))
+		glog.Infof("Image black hole in %s:\n%v", imageServiceList[1].ServiceName(), getImageListDifference(imageMapList[1], imageMapList[0]))
+	}
+
+	// Append unikernl image list at last
+	// NOTE: Here we assume all unikernel image is different from hyper/docker runtimes
```

Comment:
What does 'different' means?

Reply:
'different' means unikernel image and hyper(docker) image will never overlap, we don't need to caculate set intersection of them.